### PR TITLE
Updating cfdirectory filter with correct information

### PR DESCRIPTION
**Pull request by andrew-dixon**
The cfdirectory filter information was incorrect/incomplete, so have updated it to be correct.

### CODE REVIEW

File: data/en/cfdirectory.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		{"name":"action","description":"Action to perform","required":false,"default":"list","type":"string","values":["list","create","delete","rename"]},
 		{"name":"directory","description":"Absolute pathname of directory against which to perform\n action.","required":true,"default":"","type":"string","values":[]},
 		{"name":"name","description":"Name for output record set.","required":false,"default":"","type":"string","values":[]},
-		{"name":"filter","description":"File extension filter applied to returned names. For\n example: *.cfm. One filter can be applied.","required":false,"default":"","type":"string","values":[]},
+		{"name":"filter","description":"Filter applied to returned names. For example: *.cfm\nYou can use a pipe ("|") delimiter to specify multiple filters. For example: *.cfm|*.cfc\nFilter pattern matches are case sensitive on UNIX and Linux.","required":false,"default":"","type":"string","values":[]},
```
Comment (pfreitag) [inline data/en/cfdirectory.json]: Need to change `("|")` to `(\"|\")` to escape the quotes

**Comment by pfreitag** [comment]
Hi @andrew-dixon - thanks for the contribution, looks like the Travis CI check failed because you need to escape the double quotes in the JSON, I've put the details up above.

**Comment by andrew-dixon** [comment]
@pfreitag Fixed but the build has failed again and I can't see the error on Travis.

**Comment by pfreitag** [comment]
Ok looks like Travis CI was having an issue installing `oraclejdk8` switched it to openjdk and it seams to fix it. I'm restarting the travis build for this PR, should have a result in a few minutes

**Comment by pfreitag** [comment]
Restarting the build still wants to use the old travis.yml - the change looks good to me now so I'll just merge it. Thanks for the fix!

**Pull request merged**